docs/root/version_history/v1.14.6.rst

```diff
@@ -0,0 +1,7 @@
+1.14.6 (December 7, 2020)
+=========================
```

Comment:
do we want the blank line before Changes or no?  I'm good either way as long as we're consistent across files.

Reply:
I checked rst files for previous releases and found that some of them have blank line and others do not.
In "old" version history scheme it actually does not matter, but in "new" scheme it does. I did not put blank line in "Bug Fixes" section in 1.16.1 release notes and it lead to wrongly rendered page: https://www.envoyproxy.io/docs/envoy/v1.16.1/version_history/current. Therefore, for consistency, I suggest to put blank lines after each section. I will correct the files.